charts/kong/README.md

```diff
@@ -746,6 +749,19 @@ kong:
       TZ: "Europe/Berlin"
 ```
 
+#### The `serviceDiscovery` section
+
+With Kong Ingress Controller v2.9 a new way of configuration has been introduced
```

Comment:
this section is heavily based on the assumption that the user knows the "old" way and we're showing the "new" as a diff versus the old.

I think it would be more productive to show the two modes in an equally descriptive fashion, and make a remark that "the new one" is only valid in KIC >= 2.9